### PR TITLE
DOCS 2.4 : reactivate api: links now that api: parsed correctly

### PR DESCRIPTION
**Pull request by SpiritLevel**

### CODE REVIEW

**Comment by camfindlay** [comment]
You may need to provide some context on this @SpiritLevel - just be aware... the docs website auto pull content from the framework branches so any accepted changes here will go live overnight (even if the docs site isn't technically set up to deal with the parsing).

**Comment by SpiritLevel** [comment]
@camfindlay OK :)  All I have done is uncomment the api: links and put them back into the state they were in before beginning all of this work on the docs links. 

To get the api: links working on the docs site we need to (i) merge this PR (ii) have the docs site run the bash update command (iii) have the docs site grab the latest version of docsviewer with the new api: parser and (iv) use a stripped down .htaccess file. But, first, I imagine you want to do all of this on uat to test it.

So, once this PR is merged, can you run the bash update script on the uat server ? I'll also submit a PR for the stripped down .htaccess file that is needed for testing. Can you also put this stripped down .htaccess on uat ? Once this is done I can do the last bit of testing on the links....

File: docs/en/changelogs/2.2.0.md [hunk 1]
```diff
@@ -144,7 +144,6 @@ Sitemap.php
       * PHP OpenID (http://openidenabled.com/php-openid/)
       * Browscap (http://browscap.org/)
       * Plotr (http://www.solutoire.com/plotr/)
-      * SWFUpload (http://profandesign.se/swfupload/)
```
Comment (tractorcow) [inline docs/en/changelogs/2.2.0.md]: Don't remove items from changelogs please.

Comment (SpiritLevel) [inline docs/en/changelogs/2.2.0.md]: Oopsie....added it back....:)

**Comment by tractorcow** [comment]
Please squash and I'll merge. :)

**Comment by SpiritLevel** [comment]
Thanks Damian.

**Comment by SpiritLevel** [comment]
@tractorcow Ready to merge :)

**Pull request merged**